package throwing.custom.exception.manually;

public class Test {

	public static void main(String[] args) {
		
		throw new myException("custom exception");

	}

}
